pod.go

```diff
@@ -1162,6 +1168,44 @@ func togglePausePod(podID string, pause bool) (*Pod, error) {
 	return p, nil
 }
 
+func (p *Pod) handleDevices() error {
+	for _, device := range p.devices {
+		// Handle VFIO devices
+		// TODO Add support for handling devices besides VFIO
+
+		if device.isVFIODeviceGroup() {
```

Comment:
This is the generic group assignment. This will be uncommon but good to have. You can assign a single device in a iommu group to a VM and vfio should hold on to the other devices. So when we implement specific device assignment we should support assigning just one of the devices in a group.